### ISSUE TITLE
Missing comma in in generated FormBuilder -> buildForm()

### ISSUE THREAD

**Issue by MKL-MikaHanke** [XHAlawa/AutoFormsBuilderFilesGenerator#4]
Generated FormBuilders are missing a comma in `this.fb.group()` under certain conditions. These conditions are met when the type of an attribute of the model is another model and there is at least one more attribute behind it. The following description should clarify what i mean:

**1. Generate models with a api description like this (schemeFile):**
```
"components": {
  "schemas": {
    "Test": {
      "type": "object",
      "properties": {
        "id": { "type": "integer" },
        "testRef": { "$ref": "#/components/schemas/TestRef" },
        "propertyAfterRef": { "type": "string" }
      }
    },
    "TestRef": {
      "type": "object",
      "properties": {
        "name": { "type": "string" }
    }
  }
}
```
Important here is that `Test` has a property with a reference (`testRef`) AND another property behind/underneath it (`propertyAfterRef`).

**2. Generate the forms:**
Models should look like this:
```
export interface Test { 
    id?: number;
    testRef?: TestRef;
    propertyAfterRef?: string;
}
export interface TestRef { 
    name?: string;
}
```
The generated TestFormBuilder file should have an error, because of a missing comma:
```
@Injectable()
export class TestFormBuilder implements IFormBuilder<Test> {
    // code before ...

    buildForm(model: Test | null = null, 
        additionalControl: { name: string, control?: AbstractControl}[] = []) {
        
        this.form = this.fb.group({
			id:0, 
			testRef: [ null ] 			propertyAfterRef:'',  // HERE: missing comma after 'testRef: [ null ]'

        });

        if (model != null) {
            this.form.patchValue({ ... model });
        }

        additionalControl.forEach(a => {
          this.form.addControl(a.name, a.control ?? new FormControl())
        })

        

        return this.form;
    }

    // code after ...
}
```
---
line with error: `testRef: [ null ] 			propertyAfterRef:'',`

expected correct line: `testRef: [ null ], 			propertyAfterRef:'',`

![image](https://github.com/XHAlawa/AutoFormsBuilderFilesGenerator/assets/163290548/f0f6fe07-e401-4a30-8477-eb02caac5169)

---
I was using `"angular-formsbuilder-gen": "^1.0.50",`

**Comment by XHAlawa** [comment]
@MKL-MikaHanke   Fixed in Version ^1.0.53 
Please Check

**Comment by MKL-MikaHanke** [comment]
> @MKL-MikaHanke Fixed in Version ^1.0.53 Please Check

Yes it works, thanks for the quick fix
